example/rnn/cudnn_lstm_bucketing.py

```diff
@@ -71,29 +73,30 @@ def get_data(layout):
 
 def train(args):
     data_train, data_val, vocab = get_data('TN')
+    cells = []
```

Comment:
Since sequantialcell can do unroll this should be sequentialcell, append layers later